test/integration/index.js

```diff
@@ -16,6 +16,7 @@
 
 var _ = require('lodash');
 var chalk = require('chalk');
+var readline = require('readline');
```

Comment:
I believe this module comes with node.js: https://nodejs.org/api/readline.html, similar to http module.